client/packages/common/src/intl/locales/en/inventory.json

```diff
@@ -23,6 +23,8 @@
   "label.locked": "Locked",
   "label.batch": "Batch",
   "label.pricing": "Pricing",
+  "label.supplier": "Supplier",
+  "label.no-supplier": "Inventory Adjustment",
```

Comment:
:nit the text isn't a label, it's just text.. 
I would have `"messages.no-supplier": "Inventory Adjustment",` to fit in with the other items